release/start-kube-burner/start-kube-burner.sh

```diff
@@ -2,37 +2,68 @@
 #
 # Starts kube-burner which generates load for collector and scanner
 #
-# When running locally first have a kubeconfig file and set the following environment variables
+# When running locally first have a kubeconfig file, deploy ACS, and set the following
+# environment variables
 #
 # export KUBECONFIG=<kubeconfig file>
 # export KUBE_BURNER_CONFIG_DIR=<kube_burner_config_dir>
+# export UUID=<uuid_for_kube_burner_run>
+# export KUBE_BURNER_METRICS_FILE=<path_to_metrics_collected_by_kube_burner>
+# export METRICS_COLLECTION_TIME=<how_frequently_metrics_are_sent>
+# export METRICS_TIME_STEP=<the_time_between_metrics>
 
-set -euo pipefail
+DIR="$(cd "$(dirname "$0")" && pwd)"
 
-gh_log notice "Deploying benchmark operator"
-make -C "$BENCHMARK_OPERATOR_DIR" deploy
+KUBE_BURNER_CONFIG_DIR_BASE="$(dirname "$KUBE_BURNER_CONFIG_DIR")"
 
-# TODO(ROX-28948): When all versions using the cluster-density directory
-# are out of support, remove it here
-kube_burner_load_dir="${KUBE_BURNER_CONFIG_DIR}/berserker-load"
-if [ ! -d "$kube_burner_load_dir" ]; then
-  kube_burner_load_dir="${KUBE_BURNER_CONFIG_DIR}/cluster-density"
+# TODO(ROX-29223): Remove once old versions can use the new script
+# Test tags such as 0.0.d should be run with the new script
+if [[ "$STACKROX_VERSION" =~ ^4\.[0-7].* || "$STACKROX_VERSION" =~ ^3\.* ]]; then
+  # Don't start kube-burner for the cluster with fake data generation for older versions
+  if [[ "$LOAD_TYPE" =~ "fake" ]]; then
+    gh_log notice "Not running kube-burner for the cluster with fake workload for old version $STACKROX_VERSION"
+    exit 0
+  fi
+  export KUBE_BURNER_CONFIG_DIR="$KUBE_BURNER_CONFIG_DIR_BASE"
+  gh_log notice "Using old scripts for old version $STACKROX_VERSION"
+  "${DIR}"/old-start-kube-burner.sh
+  exit 0
 fi
 
-node_name="$(kubectl get nodes -o=jsonpath='{.items[0].metadata.name}')"
-kube_burner_cr="$KUBE_BURNER_CONFIG_DIR"/kube-burner-cr.yml
-kube_burner_cr_gen="$KUBE_BURNER_CONFIG_DIR"/kube-burner-cr-gen.yml
-
-gh_log notice "Patching $kube_burner_cr"
-sed "s|__NODE_NAME__|$node_name|" "$kube_burner_cr" > "$kube_burner_cr_gen"
+# TODO(ROX-28948): When all versions using the cluster-density directory
```

Comment:
Please mark here and in the ticket, which version is the last to use the cluster-density directory.

Reply:
Done.